### PR TITLE
Update Polish translation 200124

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

**Comment by piotrdrag** [comment]
Sure!

File: po/pl.po [hunk 1]
```diff
@@ -529,3 +533,9 @@ msgstr "Wybór urządzenia"
 #: data/ui/Window.ui:8
 msgid "_About Piper"
 msgstr "_O programie"
+
+msgid ""
```
Comment (stephanlachnit) [inline po/pl.po]: Isn't here `#: piper/window.py:55` missing? Anyway, you can rebase against master now since it's merged.

Comment (piotrdrag) [reply po/pl.po]: It doesn’t matter, the .po files will get updated during build anyway.

Comment (piotrdrag) [reply po/pl.po]: OK, rebased!

Comment (stephanlachnit) [reply po/pl.po]: Thanks, didn't know that.

**Pull request merged**